apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/DatastoreTransaction.cs

```diff
@@ -15,16 +15,15 @@
 using Google.Protobuf;
 using System;
 using System.Collections.Generic;
-using System.Threading.Tasks;
```

Comment:
Why is this section and below section changed?

Reply:
may be some line ending changes